application.go

```diff
@@ -246,6 +263,15 @@ EventLoop:
 				}
 				screen.Clear()
 				a.draw()
+			default:
+				a.RLock()
+				unknownEventHandler := a.unknownEventHandler
+				a.RUnlock()
+
+				if unknownEventHandler != nil {
+					unknownEventHandler(event)
+					continue // Don't forward event.
```

Comment:
Is this necessary?  The only effect of this seems to be skipping the chance of calling ```updater```.